unleash-yggdrasil/src/strategy_upgrade.rs

```diff
@@ -6,13 +6,43 @@ use crate::state::SdkError;
 
 const DEFAULT_STICKINESS: &str = "user_id | session_id | random";
 
+enum StrategyType {
+    Default,
+    UserWithId,
+    GradualRolloutUserId,
+    GradualRolloutSessionId,
+    GradualRolloutRandom,
+    FlexibleRollout,
+    RemoteAddress,
+    Custom(String),
+}
```

Comment:
Is this accessible somehow from the Engine (or maybe the is_custom function that provided a strategy name, it returns true or false whether its custom or not)? I'm thinking of a centralized list of strategies. Maybe something for later

Reply:
Yuh, thought about it too, it's good feedback. Currently not accessible but we can make it so in the future. Probably want to do that in another PR though, this one is too big already